Typos; small iset-FOL restructuring

benjub opened the pull request:
* iset and mmil.raw.html: ax-12 --> ax12 since it it a $p-statement, not an $a-statement. If we want consistency with set.mm labels, then we can create an alias ax12 of ax-12 in set.mm (and rename the current ax12 to ax12from12v).
* iset.mm: since ax12or exists, use it consistently and discourage ax-i12.
* iset.mm: move wcel/wel to the end of FOL for clearer structure.
* iset.mm: typography fixes
* set.mm: mainly typography, and small rewordings (fyi @tirix @avekens in your mathboxes).

icecream17 commented on set.mm:
```diff
@@ -748370,9 +748375,9 @@ be a Fermat number (i.e., a Fermat prime), see ~ 2pwp1prmfmtno .
-    $( For every prime number of the form ` ( ( 2 ^ k ) + 1 ) ` ` k ` must be a
-       power of 2, see Wikipedia "Fermat number", section "Other theorms about
-       Fermat numbers", ~ https://en.wikipedia.org/wiki/Fermat_number ,
+    $( For the positive integer ` ( ( 2 ^ k ) + 1 ) ` to be prime, ` k ` must
+       be a power of 2, see Wikipedia "Fermat number", section "Other theorms
```
```suggestion
    $( If ` ( ( 2 ^ k ) + 1 ) ` is prime for positive integer ` k ` , ` k ` is
       a power of 2, see Wikipedia "Fermat number", section "Other theorms
```

assuming I'm interpreting the intent of this change correctly, a fix for the case where $$2^{\log_2{6}} + 1$$ is technically a prime positive integer could be like this

icecream17 replied:
After more thinking, this is probably not the intent. You may optionally exclude ``for positive integer ` k ` `` since that's rather inherent/obvious, or simply remove `the positive integer` from the start (primes are positive integers so it's redundant) instead of doing my suggestion

benjub replied:
I'll do the latter. My concern was purely English style.

icecream17 commented on iset.mm:
```diff
@@ -287,9 +287,9 @@ described in the Metamath book (or the online HELP).
-mmj2 program will justify the definitions as sound, except for 5 (df-bi,
-df-cleq, df-clel, df-clab, df-sbc) that require a more complex metalogical
-justification by hand.
+mmj2 program will justify the definitions as sound, except for five of them
+(df-bi, df-cleq, df-clel, df-clab, df-sbc) that require a more complex
+metalogical justification.
```
It doesn't seem like df-sbc is an exception:

https://github.com/metamath/set.mm/blob/e50dd75c9a8fb079cc6dbb3620b787e247699561/.github/workflows/verifiers.yml#L289

but I'm probably missing something

benjub replied:
Actually, I blindy copied the analogous part from set.mm. So one of them has to be fixed. Maybe I "fixed" the wrong one... I'm going to check.

benjub replied:
Done: you were right, actually iset.mm was correct and it is set.mm that had to be fixed. (Well, the correct fix would be to simply have set.mm and iset.mm share that information, by including a common file. That's still in my plans, for later.)

icecream17 reviewed (commented):
LGTM though I skimmed, and there might be future commits, but submitting these early since you are currently making commits so it's easier to incorporate

avekens commented on set.mm:
```diff
@@ -17804,32 +17805,32 @@ disjoint variable condition ( ~ sb2 ).  Theorem ~ sb6f replaces the
-     means that the predicate has two arguments.
-
-     (Instead of introducing ~ wel as an axiomatic statement, as was done in an
+     symbol ` e. ` into our predicate calculus.  We will eventually use it for
```
Why did you remove "(stylized lowercase epsilon)"?

benjub replied:
It's in the comment of the $c-statement introducing it.  For most other symbols, there is no description of what they look like (or only in a comment of the $c-statement).  It is displayed in the HTML page, for people to see what it looks like.

That said, I can put it back, though the addition looks a bit out of place to me.

avekens replied:
Since the comment for the  $c-statement is not dsiplayed in the HTML page, I would prefer to have this put back.

benjub replied:
Change reverted.

avekens commented on set.mm:
```diff
@@ -101234,10 +101237,9 @@ strictly dominates its argument (or the supremum of all ordinals that
-     ` aleph `` x = x ` .  (Contributed by NM, 6-Nov-2004.)  (Revised by Mario
-     Carneiro, 15-May-2015.) $)
+     point.  Compare the inequality ~ alephle that holds in general.
+     (Contributed by NM, 6-Nov-2004.)  (Revised by Mario Carneiro,
```
Why did you remove "Note that if ` x ` is a fixed point, then ` aleph `` aleph `` aleph `` ` ... ` aleph `` x = x ` ."?

avekens replied:
I do not know Norm's intention of this, but probably had one. Since it is not wrong, I would like to keep it.

benjub replied:
Change reverted.

avekens commented on set.mm:
```diff
@@ -148815,8 +148817,8 @@ notation is used by Donald Knuth for iterated exponentiation (_Science_
 
-    $( ` ``' G ` maps the size function's value to ` card ` .  (Contributed by
-       Paul Chapman, 22-Jun-2011.)  (Revised by Mario Carneiro,
+    $( The function ` ``' G ` maps the size function's value to ` card ` .
```
Then "The converse of function ` G ` ..."

benjub replied:
Done. The reason of that edit was to avoid starting a sentence with a symbol.

avekens commented on set.mm:
```diff
@@ -427289,8 +427290,8 @@ All axiomatic assertions ($a statements)
-       except for 4 definitions ( ~ df-bi , ~ df-cleq , ~ df-clel , ~ df-clab )
-       that require a more complex metalogical justification by hand.
+       except for four of them ( ~ df-bi , ~ df-clab , ~ df-cleq , ~ df-clel )
+       that require a more complex metalogical justification.
```
Why did you remove "by hand"? How atre they justified otherwise?

benjub replied:
I think "by hand" adds nothing to the sentence. (Also, that "more complex metalogical justification" could be made by a human, or by a more complex algorithm in the future, or be formalized in a "metamath-in-metamath" database or in Mario's metamath0... So, removing "by hand" is more about being agnostic about how that justification may be done.)

avekens replied:
But how are the justified currently, i.e., within or at least for set. mm? If there were not justified yet, most of set.mm would be meaningless.

icecream17 replied:
I suppose to be more specific, we can add a link explaining the current metalogical justification in more detail

benjub replied:
The justification is in Lévy's Basic Set Theory, Appendix, as written in the homepage mmset.html and discussed on the google discussion group multiple times.

Anyway, I'll revert my change since I don't want to spend more time on two words.

avekens commented on set.mm:
```diff
@@ -764894,19 +764899,22 @@ According to Wikipedia ("Ackermann function", 8-May-2024,
+  ~ ackval2012 : ` A ( 2 , 0 ) = 3 ` , ` A ( 2 , 1 ) = 5 ` ,
+                 ` A ( 2 , 3 ) = 7 ` ;
+  ~ ackval3012 : ` A ( 3 , 0 ) = 5 ` , ` A ( 3 , 1 ) = ; 1 3 ` ,
+                 ` A ( 3 , 3 ) = ; 2 9 ` .
```
Why did you insert line breaks? I thik they are not helpful at this place.

benjub replied:
I agree it's a pity, but after I added the spaces between the backtick and the comma, the lines were 80-character wide, which is more than the 79 maximum.

avekens replied:
OK, knowing the reason now I can accept this.

icecream17 commented on set.mm:
```diff
@@ -769972,7 +769980,7 @@ _modus ponens_ (our ~ ax-mp ), but he later qualifies this.
 $(
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-  Formally define terms such as Reflexivity
+  Formally define notions such as reflexivity
```
(optional, unrelated suggestion)
add ` ~ ` before the link at the bottom of this commit
![image](https://github.com/user-attachments/assets/60c342ad-6403-43bb-9c4f-8fcbf0e55196)

benjub replied:
Done.

icecream17 commented on mmnotes.txt:
```diff
@@ -4890,7 +4890,7 @@ http://planetx.cc.vt.edu/AsteroidMeta/metamath ).  It has actually been
 This eliminates the need for the Axiom of Replacement, from which
 empty set existence is derived.  The original can be seen at
-http://de2.metamath.org/metamath/set.mm .
+https://de.metamath.org/metamath/set.mm .
```
It seems like the new link doesn't work

benjub replied:
Both old and new versions are broken... It's probably lost forever, and not a big loss. I'd simply remove that sentence, but this is in old notes anyway. I'd leave it like that in this MR.

jkingdon replied:
Sounds like it is in the git history somewhere, if I'm correctly following. But it doesn't seem especially important to have a specific link in `mmnotes.txt` so just removing the sentence seems fine (at least for now, probably indefinitely).

benjub commented:
I reverted both controversial changes, that is, I restored
> Note that if ` x ` is a fixed point, then ` aleph `` aleph `` aleph `` ` ... ` aleph `` x = x ` .

and changed back
> metalogical justification.

to
>metalogical justification by hand.

avekens commented:
> I reverted both controversial changes, that is, I restored
> 
> > Note that if `x` is a fixed point, then `` aleph `` aleph `` aleph `` `` ... `` aleph `` x = x `` .
> 
> and changed back
> 
> > metalogical justification.
> 
> to
> 
> > metalogical justification by hand.

Sorry, @benjub, for my possibly annoying comments, but I often hesitate and feel uncertain when information provided by Norm is removed. I always fear that we destroy  some part of his legacy. And unfortunately, we do not know about his intention to provide this information, and, alas, we cannot ask him anymore.

jkingdon commented:
> I often hesitate and feel uncertain when information provided by Norm is removed. I always fear that we destroy some part of his legacy.

I tend to agree with this, at least as a starting point.

I also tend to default to the assumption that it is worth maintaining Norm's style.

I do not take either of these as absolutes - this isn't a static project and Norm never treated it as such when he was involved - but I do hope that we manage to preserve the good parts of what we were left.
